error when starting up: `make failed`

Hello, 
Thank you for this extension. I'm getting the following errors when I installed it and then started up the ddev container:
```
3.918 libtool: link: cc -shared  -fPIC -DPIC  .libs/oci8.o .libs/oci8_lob.o .libs/oci8_statement.o .libs/oci8_collection.o .libs/oci8_interface.o .libs/oci8_failover.o   -L/usr/local/instantclient -lclntsh  -g -O2 -Wl,-rpath -Wl,/usr/local/instantclient   -Wl,-soname -Wl,oci8.so -o .libs/oci8.so
3.921 /usr/bin/ld: skipping incompatible /usr/local/instantclient/libclntsh.so when searching for -lclntsh
3.921 /usr/bin/ld: cannot find -lclntsh: No such file or directory
3.921 /usr/bin/ld: skipping incompatible /usr/local/instantclient/libclntsh.so when searching for -lclntsh
3.921 collect2: error: ld returned 1 exit status
3.921 make: *** [Makefile:232: oci8.la] Error 1
3.925 ERROR: `make' failed
------
', stderr='failed to solve: process "/bin/bash -c echo 'instantclient,/usr/local/instantclient' | pecl install oci8" did not complete successfully: exit code: 1'
```

These are only the last few lines of the error output. I can provide more if that's helpful. 

**DDEV VERSION**: 1.24.4
**HOST OS**: Mac

--- a/docker-compose.oci8.yaml
+++ b/docker-compose.oci8.yaml
@@ -1,0 +1,7 @@
+#ddev-generated
+services:
+  web:
+    # The platform statement here has no effect on AMD64/X86 systems
+    # On ARM64 (macOS Apple Silicon) it will make it work if Rosetta2 is working. The entire
+    # web container becomes amd64.
+    platform: linux/amd64

--- a/install.yaml
+++ b/install.yaml
@@ -1,7 +1,5 @@
 # Details about the install.yaml file are at https://ddev.readthedocs.io/en/stable/users/extend/additional-services/#sections-and-features-of-ddev-get-add-on-installyaml
-
 name: oci8
-version: 1.0.0
 
 # pre_install_actions - list of actions to run before installing the addon.
 # Examples would be removing an extraneous docker volume,
@@ -60,13 +58,10 @@ pre_install_actions:
 # If you use directories, they must be directories that are managed
 # by this add-on, or removal could remove things that are not owned by it
 project_files:
+  - docker-compose.oci8.yaml
   - php/oci8.ini
   - web-build/Dockerfile.oci8
   - web-build/install-oci8.sh
-  # - some-directory/file1.txt
-  # - some-directory/file2.txt
-  # - extra_files_dir_created_by_this_template/
-  # - somefile.sh
 
 # List of files and directories that are copied into the global .ddev directory
 # DDEV environment variables can be interpolated into these filenames
@@ -96,9 +91,6 @@ post_install_actions:
 # Files listed in project_files section will be automatically removed here if they contain #ddev-generated line.
 # removal_actions are executed in the context of the target project's .ddev directory.
 removal_actions:
-  -  rm -f ${DDEV_APPROOT}/.ddev/web-build/install-oci8.sh
-  -  rm -f ${DDEV_APPROOT}/.ddev/web-build/Dockerfile.oci8
-  -  rm -f ${DDEV_APPROOT}/.ddev/php/oci8.ini
 
   # - rm ~/.ddev/commands/web/somecommand
   # - |

--- a/php/oci8.ini
+++ b/php/oci8.ini
@@ -1,2 +1,3 @@
+#ddev-generated
 [PHP]
 extension=oci8.so

--- a/web-build/Dockerfile.oci8
+++ b/web-build/Dockerfile.oci8
@@ -1,8 +1,10 @@
+#ddev-generated
+
 ARG BASE_IMAGE
 FROM $BASE_IMAGE
 
 # Oracle instantclient
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y -o Dpkg::Options::="--force-confnew" --no-install-recommends --no-install-suggests gcc make autoconf libc-dev pkg-config php-pear php8.2-dev libmcrypt-dev build-essential libaio1 unzip wget
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y -o Dpkg::Options::="--force-confnew" --no-install-recommends --no-install-suggests gcc make autoconf libc-dev pkg-config php-pear php${DDEV_PHP_VERSION}-dev libmcrypt-dev build-essential libaio1 unzip wget
 
 # Download Oracle Instant Client packages
 RUN wget https://github.com/takielias/oracle-instantclient/raw/main/instantclient-basic-linux.x64-12.1.0.2.0.zip -O /tmp/instantclient-basic-linux.x64-12.1.0.2.0.zip && \

--- a/web-build/install-oci8.sh
+++ b/web-build/install-oci8.sh
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+#ddev-generated
+
 # Exit immediately if a command exits with a non-zero status.
 set -e
 

